src/styles/main.scss

```
@@ -0,0 +1,80 @@
+@import 'reset';
+@import 'values';
+
+html {
+  font-family: Arial, Helvetica, sans-serif;
+}
+
+.calendar {
+  display: flex;
+  flex-wrap: wrap;
+  max-width: 707px;
+  gap: 1px;
+
+  &__day {
+    position: relative;
+    box-sizing: border-box;
+    border: 1px solid black;
+    width: 100px;
+    height: 100px;
+    color: #eee;
+    display: none;
```

This violates checklist item #7: 'Use SCSS variables for all repeated values and calculations.' The value 'display: none;' is set by default for all days, but only days after the month length should be hidden. Consider setting display: block by default and hiding extra days via the month-length modifier.